src/app/functions/server/getRelevantGuidedSteps.ts

```diff
@@ -244,17 +265,23 @@ export function hasCompletedStepSection(
 export function hasCompletedStep(
   data: StepDeterminationData,
   stepId: StepLink["id"],
+  enabledFeatureFlags?: FeatureFlagName[],
 ): boolean {
-  // TODO: MNTOR-3880 Waiting for criteria for data brokers under maintenace to be determined
-  // if (stepId === "DataBrokerManualRemoval") {
-  //   return (
-  //     data.latestScanData?.results?.every(
-  //       (result) =>
-  //         result.broker_status !== "removal_under_maintenance" ||
-  //         result.manually_resolved,
-  //     ) ?? false
-  //   );
-  // }
+  /* c8 ignore start */
+  // Already covered by unit tests
```

Comment:
This is likely related to the broken dashboard stories. We should not need the ignore comment.

Reply:
Ok nvm — seems like the broken story was not the only issue.